examples/enable-persistence-queue/README.md

```diff
@@ -0,0 +1,8 @@
+# Example of chart configuration
+
+## Enable Persistent Queue for logs, metrics and traces
+
+This example will show how to enable data persistence for log/metric/trace data. Data will be persisted in node's local filesystem. 
+Persistent queue will keep track of unexported data. It will continue from previously saved offsets, if any, after collector restarts and export it to Splunk platform.
+
+Refer to: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#data-persistence
```

Comment:
I'm not sure if this example is needed. I'd rather put more comments in the other example. e.g.

```
# If you want to disable the persistent queue in a particular pipeline, you can do so using the following config
# update splunk_hec/platform_traces exporter to use memory queue instead
agent:
  config:
    exporters:
       splunk_hec/platform_traces:
         sending_queue:
           storage: null
```

Reply:
yeah, we can do that. but I added one for enabling and one for disabling, just in case.

Reply:
Added support for [no-render](https://github.com/signalfx/splunk-otel-collector-chart/blame/main/examples/DEVELOPER_GUIDE.md) examples which could be useful for examples like this in the future.